### PR TITLE
css for nav

### PR DESCRIPTION
**Pull request by tyeung2017**
relates #23 #32 

improved css for nav

### CODE REVIEW

**Review by KatiaAsh — APPROVED** [review]

**Review by Wyrd00 — APPROVED** [review]

**Comment by Karyum** [comment]
before i review you need to relate to an issue in the PR description

**Comment by Karyum** [comment]
also in the commits for the future

**Comment by Karyum** [comment]
I don't understand this PR cause there is no issue for it.
is it building the nav bar or is it styling it or what

**Comment by Wyrd00** [comment]
it's both styling and htmling, friend 🤗

**Comment by Karyum** [comment]
@tyeung2017 also you can't just make a commit which the message is literally just "nav" please don't let that happen in the future. add a useful message

File: images/head.svg [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```
Comment (Karyum) [inline images/head.svg:60]: whats going on here why so many g tags?

File: images/user.svg [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
```
Comment (Karyum) [inline images/user.svg:40]: also here

File: images/head.svg (hunk 1 above)
Comment (tyeung2017) [reply images/head.svg:60]: no sure about that 

we just get this from some online tools found by Amy

**Comment by Karyum** [comment]
untitled

is this how you want it to look like ?

**Comment by tyeung2017** [comment]
yeah just a placeholder

Or you have better suggestion?

**Comment by Karyum** [comment]
@tyeung2017 i mean it's positioned weirdly if thats how you want it look like it's fine then

**Comment by KatiaAsh** [comment]
@Karyum Yes, can you merge it now? :thinking:

**Pull request merged**